src/Core/Style.js

```diff
@@ -160,6 +161,64 @@ function defineStyleProperty(style, category, name, value, defaultValue) {
     style[category][name] = value;
 }
 
+/**
+ * @class
+ * @classdesc StyleContext stores metadata of one FeatureGeometry that are needed for its style computation:
+ * type of feature and what is needed (fill, stroke or draw a point, etc.) as well as where to get its
+ * properties and its coordinates (for base_altitude).
+ *
+ * @property {Coordinates}      worldCoord @private Coordinates of the FeatureGeometry in world system.
+ * @property {Coordinates}      localCoordinates @private Are the coordinates systeme origin local or global.
+ * @property {boolean}          isProjected @private Are the coordinates already been projected.
+ * @property {FeatureGeometry}  geometry  @private
+ * @property {Object}           globals Style type (fill, stroke, point, text and or icon) to consider. As
+ *                                  well as the zoom.
+ * @property {Object}           collection The FeatureCollection on which the FeatureGeomatry is attached
+ * @property {Coordinates}      coordinates The coordinates (in world crs) of the last vertice set with
```

Comment:
* `in world crs` -> `in world space` because the code suggests that the crs is the same as the crs of `localCoordinates`.
* `vertice` -> `vertices`

Reply:
A vertice is the combination of 3 informations: x, y and z, if i'm not mistaken. get coordinates() get us information of the one vertice that we did set.

Reply:
you're right there is only one, then it's `vertex`